docs/source/installing_docker.rst

```diff
@@ -79,6 +79,28 @@ You need to create the database schema by executing::
         docker exec -it kiwi_web /Kiwi/manage.py createsuperuser
 
 
+.. _configure-kiwi-base-url:
+
+Configuration of Kiwi's base URL
+--------------------------------
+
+The first step you need to do is configure the base URL of your Kiwi TCMS
+installation. This is used to construct links to test plans, test cases, etc.
+The default value is ``127.0.0.1:8000`` which is suitable if you are running
+in devel mode. The easiest and automation-friendly way to set base URL is
+to use command line tool::
+
+    ./manage.py baseurl https://public.tenant.kiwitcms.org
```

Comment:
Domain, not URL

Reply:
It might be a good idea to rename the command too? 
./manage.py site or ./manage.py domain?
The help text will be: Sets the site / domain of Kiwi TCMS instance.

Reply:
Use `set_domain`